crypto/keyring/entry.go

```diff
@@ -8,28 +8,55 @@ import (
 	"github.com/cosmos/cosmos-sdk/types"
 )
 
-var (
-	_ Info = &KeyringEntry{}
-)
 
-// Info is the publicly exposed information about a keypair
-type Info interface {
```

Comment:
Since we removed localInfo, offlineInfo structs, I would like to replace Info interface with *KeyringEntry from [types.pb.go](https://github.com/cosmos/cosmos-sdk/blob/kerying-migration/crypto/keyring/types.pb.go#L27).  Do you think it makes sense ? cc @robert-zaremba   @aaronc  @amaurym

Reply:
I'm okay to remove the `Info` interface. Since we decided to use the `KeyringEntry` struct, we don't need this interface anymore for the proto keyring.

We probably need to rename this interface to LegacyInfo for managing amino migration.